modules/reindex/src/main/java/org/elasticsearch/reindex/UpdateByQueryMetrics.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.reindex;
+
+import org.elasticsearch.telemetry.metric.LongHistogram;
+import org.elasticsearch.telemetry.metric.MeterRegistry;
+
+public class UpdateByQueryMetrics {
+    public static final String TOOK_TIME_HISTOGRAM = "es.update_by_query.took_time.histogram";
```

Comment:
maybe we should name this metric `es.update_by_query.duration.histogram`? instead of using `took_time`?

Reply:
Yes sure.